### ISSUE TITLE
Preference to block third-party images

### ISSUE THREAD

**Issue by jonathansampson** [brave/browser-laptop#6875]
As mentioned [in this comment](https://github.com/brave/browser-laptop/commit/4c30cd08af9dda75a0e3d6ef634abbbdc2a39c74#commitcomment-17570223), and [requested elsewhere](https://gab.ai/judgedread/posts/4497473), users have requested third-party image blocking.

**Comment by flamsmark** [comment]
I don't think that blocking 3rd-party *images* specifically — as opposed to 3rd-party content in general, or cookies on 3rd-party content, or… — is a privacy issue.

**Comment by bsclifton** [comment]
@flamsmark what about content fetched which includes a tracking ID?

ex: `http://example/tracking.jpg?id=123456`

**Comment by jonathansampson** [comment]
It's definitely not a major concern, but could carry query parameters and/or your ip address off to unrelated parties. I'm not personally worried about this, but can see how others might be.

**Comment by flamsmark** [comment]
@bsclifton Is the tracking implication different when it's `https://example/tracking.jpg?id=123456` as opposed to `http://example/tracking.txt?id=123456`. That is: what's the relationship between the privacy consequences and the filetype being requested?

My preference is to combine this with other [content-blocking](https://github.com/brave/browser-laptop/labels/feature%2Fcontent-blocking) requests into one integrated feature. I'm not sure what relationship that will have with the other Shields UI. Does that sound sensible, or is there good reason to work on this separately?

**Comment by bsclifton** [comment]
@flamsmark good call! Having a catch-all (regardless of file type) sounds great 😄
